src/main/scala/ai/eto/rikai/sql/spark/execution/DropModelCommand.scala

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2021 Rikai authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package ai.eto.rikai.sql.spark.execution
+
+import ai.eto.rikai.sql.model.{Catalog, ModelNotFoundException}
+import org.apache.spark.sql.execution.command.RunnableCommand
+import org.apache.spark.sql.{Row, SparkSession}
+
+case class DropModelCommand(
+    name: String
+) extends RunnableCommand {
+
+  override def run(session: SparkSession): Seq[Row] = {
+    val catalog =
+      Catalog.getOrCreate(
+        session.conf.get(
+          Catalog.SQL_ML_CATALOG_IMPL_KEY,
+          Catalog.SQL_ML_CATALOG_IMPL_DEFAULT
+        )
+      )
+    if (!catalog.dropModel(name))
+      throw new ModelNotFoundException(s"Model not found: ${name}")
```

Comment:
Could this fail due to reasons other than the model not being found?

Reply:
If so, `dropModel` can throw exceptions (i.e., network error or sth) as well.